spec/controllers/conference_registration_controller_spec.rb

```diff
@@ -0,0 +1,284 @@
+require 'spec_helper'
+
+describe ConferenceRegistrationsController, type: :controller do
+  let!(:first_user) { create(:user) }
+  let(:conference) { create(:conference) }
+  let(:conference_with_open_registration) { create(:conference) }
+  let!(:open_registration_period) { create(:registration_period, conference: conference_with_open_registration, start_date: Date.current - 6.days) }
+  let(:user) { create(:user) }
+
+  context 'user is not signed in' do
+    describe 'GET #new' do
+      context 'ichain is enabled' do
```

Comment:
Do we need to cover what happens for `GET #new` when the user is not signed in and ichain is not enabled?